app/reducers/device/status_bar.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {DeviceTypes} from 'app/constants';
+
+export default function height(state = 20, action) {
```

Comment:
Isn't this default dependent on the platform?

Reply:
This was basically a copy paste of the old code, so I guess it doesn't matter much but I can change it to be 0